userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -92,6 +103,10 @@ _Example:_
 
 |===
 
+External Payment::  A built-in payment method that can be used to track payments that occur outside of Kill Bill.
```

Comment:
This seems like it needs another sentence, like where it's set in the system. I started to look into it and was surprised to see a reference to an "EXTERNAL_PAYMENT plugin" in the Payment Processing Guide. Is there really such a thing?

Reply:
I don't think this is correct or clear: An external payment is a payment that was created using an external payment method (e.g check). But currently the sentence mixes payment and payment method.

Reply:
I've changed the term from `External Payment` to `External Payment Method` (which is what I originally had in mind). LMK if this makes sense now.